### PR TITLE
Controllers

### PR DESCRIPTION
**Pull request by annchemu**
@drizzentic  please review and merge

### CODE REVIEW

File: app/Http/Controllers/StockController.php [hunk 1]
```diff
@@ -0,0 +1,286 @@
...
+            // }
+            // else if($usage->quantity_used>Topup::find((int)$usage->request_id)->quantity_ordered)
+            // {
+            //     return Redirect::back()->with('message', trans('messages.issued-greater-than-ordered'))->withInput(Input::all());
+            // }
+            // else
+            // {
+            //     $usage->save();
+            
+            //     return Redirect::to($url)->with('message', trans('messages.record-successfully-updated'))->with('active_stock', $usage->stock->id);
```
Comment (drizzentic) [inline app/Http/Controllers/StockController.php]: Get rid of the commented code

File: routes/web.php [hunk 2]
```diff
@@ -15,3 +15,20 @@
...
+Route::resource('test', 'TestController');
+Route::resource('specimen', 'SpecimenController');
+Route::resource('testtype', 'TesttypeController');
+Route::resource('report', 'ReportController');
+Route::resource('inventory', 'InventoryController');
+Route::resource('supplier', 'SupplierController');
+Route::resource('item', 'ItemController');
+Route::resource('facility', 'FacilityController');
+Route::resource('drug', 'DrugController');
+Route::resource('critiical', 'CritiicalController');
```
Comment (drizzentic) [inline routes/web.php]: Typo. lol!

**Review by drizzentic — CHANGES_REQUESTED** [review]
have a look

**Review by briankip — CHANGES_REQUESTED** [review]
Dont delete .ev.example

**Comment by annchemu** [comment]
@drizzentic check it out.

**Review by briankip — APPROVED** [review]

**Pull request merged**